src/plugins/embeddable/public/lib/embeddables/error_embeddable.tsx

```diff
@@ -44,16 +49,28 @@ export class ErrorEmbeddable extends Embeddable<EmbeddableInput, EmbeddableOutpu
     } catch (err) {
       theme = {};
     }
-    const node = (
+    const errorMarkdown = (
+      <Markdown markdown={title} openLinksInNewTab={true} data-test-subj="errorMessageMarkdown" />
+    );
+
+    const node = this.compact ? (
+      <EuiFlexGroup
+        style={{ height: '100%', whiteSpace: 'nowrap' }}
+        gutterSize="none"
```

Comment:
Perhaps `gutterSize="xs"` might be a little better here?

![image](https://user-images.githubusercontent.com/8698078/172210595-6086e354-a01a-4ce2-903a-1c8cc2633565.png)